### ISSUE TITLE
tf.contrib.layers.l2_regularizer() cause warning

### ISSUE THREAD

**Issue by carltonwang** [tensorflow/tensorflow#18151]
TF version: 1.7
```
import tensorflow as tf
r = tf.contrib.layers.l2_regularizer(0.001)
```
Then a warning message will occur:

> WARNING:tensorflow:From /.../tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
> Instructions for updating:
> Use the retry module or similar alternatives.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @cy89: It has been 75 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by carltonwang** [comment]
issue solved in TF 1.8
